.npmrc

```diff
@@ -0,0 +1,2 @@
+engine-strict=true
```

Comment:
intended?

Reply:
yeah its intended, have the same thing in xmint main